inlong-manager/manager-web/src/main/resources/application.properties

```diff
@@ -64,3 +64,11 @@ openapi.auth.enabled=false
 # audit view by role, see audit id definitions: https://inlong.apache.org/docs/modules/audit/overview#audit-id
 audit.admin.ids=3,4,5,6,7,8
 audit.user.ids=3,4,5,6,7,8
+
+# data cleansing
+# if turned on, logically deleted group data will be collected and permanently deleted periodically
+group.purge.enabled=false
```

Comment:
This deletion operation is not limited to the inlong group, but also inlong stream, source, sink, etc.

Therefore, the names of the config params should be common.